pkg/webhooks/policy/handlers.go

```diff
@@ -52,6 +53,14 @@ func (h *policyHandlers) Validate(ctx context.Context, logger logr.Logger, reque
 		return admissionutils.Response(request.UID, err, warnings...)
 	}
 
+	if gpol := policy.AsGeneratingPolicy(); gpol != nil {
+		warnings, err := gpolvalidation.Validate(gpol)
+		if err != nil {
+			logger.Error(err, "generating policy validation errors")
+		}
+		return admissionutils.Response(request.UID, err, warnings...)
```

Comment:
We no longer deny the policy admission upon failures/errors, we should use policy status instead to indicate its readiness.

Reply:
I see that we block validatingpolicies in case there are compiler errors or missing matchconstraints.
https://github.com/kyverno/kyverno/blob/c55de7fa21323952ac6f912e38030756bd73cbd0/pkg/cel/policies/vpol/validate.go#L27-L31